src/chainparamsbase.cpp

```diff
@@ -9,6 +9,7 @@
 #include "util.h"
 
 #include <assert.h>
+#include <string.h>
```

Comment:
this looks unrelated...

Reply:
Weird.. `string.h` is included at least in `util.h`, so should be compiling just fine imo.

Reply:
Ok I'll remove that include then.